src/artefacts_demo_control/world/artefacts_demo_world.sdf

```diff
@@ -2518,13 +2518,123 @@
         <robot_param_node>robot_state_publisher</robot_param_node>
         <parameters>/home/decarabas/artefacts_interbotix_movit2/install/interbotix_xsarm_sim/share/interbotix_xsarm_sim/config/trajectory_controllers/wx200_trajectory_controllers.yaml</parameters>
       </plugin>
+
+      <plugin name="gazebo_ros_state" filename="libgazebo_ros_state.so">
```

Comment:
@MDecarabas are you using this plugin? if no better to remove it

Reply:
done